### ISSUE TITLE
Required fields are not announced by screen reader

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#2915]
## Description
Any required fields are not announced by a screen reader when navigating forms.

### Related

- https://github.com/GCTC-NTGC/gc-accessibility-passport/issues/111

**Comment by tristan-orourke** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/GC-Digital-Talent-6092dc05157fa30010edaef9/issues/GCTC-NTGC/gc-digital-talent/2915?planning-poker) with ZenHub @esizer

**Comment by esizer** [comment]
The work should be low effort but tedious, hence the 2.

Should just need to add `required` attribute to all of our inputs.

```tsx
<Input required />
```

**Comment by mnigh** [comment]
> The work should be low effort but tedious, hence the 2.
> 
> Should just need to add `required` attribute to all of our inputs.
> 
> ```tsx
> <Input required />
> ```

You might run in to some problems there (as noted [here](https://github.com/GCTC-NTGC/gc-accessibility-passport/pull/159)). Adding an `aria-required` attribute was the solution I came up with.

**Comment by esizer** [comment]
> You might run in to some problems there (as noted [here](https://github.com/GCTC-NTGC/gc-accessibility-passport/pull/159)). Adding an `aria-required` attribute was the solution I came up with.

Yikes, that is not good 😢 . Maybe we can bump up the estimate to see if we can get `required` working to avoid using `aria-required="true"`?

**Comment by tristan-orourke** [comment]
Let's go with ariea-required in our input components for now.
